src/main/java/com/requestrealpiano/songrequest/domain/letter/Letter.java

```diff
@@ -0,0 +1,50 @@
+package com.requestrealpiano.songrequest.domain.letter;
+
+import com.requestrealpiano.songrequest.domain.account.Account;
+import com.requestrealpiano.songrequest.domain.song.Song;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+public class Letter {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "letter_id")
+    private Long id;
+
+    @Column(name = "song_story")
+    private String songStory;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "request_status")
+    private RequestStatus requestStatus;
+
+    @Column(name = "created_date_time")
```

Comment:
```suggestion
    @Column(name = "created_date")
```
p3 : 같은 역할을 하는 컬럼들의 이름은 통일되는 게 좋을 것 같아요

Reply:
- https://velog.io/@conatuseus/2019-12-06-2212-%EC%9E%91%EC%84%B1%EB%90%A8-1sk3u75zo9
- p4 : 이런 기능도 한번 생각해보셨으면 좋을 것 같아요

Reply:
이전에 날짜만 다루는 컬럼이나 필드에는 Date, 시간도 함께 다루는 것에는 DateTime으로 네이밍을 구분하는게 좋다는 피드백을 받은적이 있었습니다. 다만 @102092 한이 지적 해주신 것처럼 통일성이 지켜지지 않았네요. Account에 있는 관련 필드를 created_date_time으로 수정 하는게 좋을거 같습니다. 놓쳤던 부분인데 감사합니다!

Reply:
이전에 날짜만 다루는 컬럼이나 필드에는 Date, 시간도 함께 다루는 것에는 DateTime으로 네이밍을 구분하는게 좋다는 피드백을 받은적이 있었습니다. 다만 @102092 한이 지적 해주신 것처럼 통일성이 지켜지지 않았네요. `Account` 에 있는 관련 필드를 `created_date_time` 으로 수정 하는게 좋을거 같습니다. 놓쳤던 부분인데 감사합니다!

Reply:
JPA Auditing은 한번도 깊게 살펴보지 않았던 기능인데 이번 프로젝트에서 공부하고 적용 해보겠습니다.

Reply:
JPA Auditing은 한번도 깊게 살펴보지 않았던 기능인데 이번 프로젝트에서 공부하고 적용 해보겠습니다 👍